src/client/components/ModalNew.tsx

```diff
@@ -157,3 +157,26 @@ export const Modal: React.FC<ModalProps> = ({
     </Wrapper>
   )
 }
+
+const Footer = styled.div<ModalFooterProps>`
+  padding: 1rem 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  ${({ sticky }) =>
+    !sticky &&
+    `
+    box-shadow: 0px -4px 8px rgba(0, 0, 0, 0.05), 0px -8px 16px rgba(0, 0, 0, 0.05);
+    background: ${colorsV3.white};
+    position: sticky;
+    bottom: 0;
+`}
+`
+type ModalFooterProps = {
+  sticky?: boolean
+}
+
+export const ModalFooter: React.FC<ModalFooterProps> = ({ children }) => {
+  return <Footer>{children}</Footer>
+}
```

Comment:
Nit: we avoid using `React.FC`. You can read more here: https://www.notion.so/hedviginsurance/React-components-props-f13a8f7ebfae4ee0b57fe47b55b6a156#7ad1d210a58a4ea4a4814c3b682e13be

At least for new code :)